### DISCUSSION TITLE
Theme of Digital Garden Docs?

### DISCUSSION THREAD

**Discussion by andrey-jef** [oleeskild/obsidian-digital-garden#232] [Q&A] [answered]
Hi,

The appearance (font + color) of the [Digital Garden Docs](https://dg-docs.ole.dev/) is beautiful. Is it possible to replicate this view by changing the name of theme in the setting of Digital Garden plugin?

Thanks

**Comment by oleeskild** [comment, marked as answer]
Hi!
It's using the Red Graphite theme with the "Dark" base theme. It has no custom CSS.
